cc/move.cc

```diff
@@ -0,0 +1,31 @@
+// Copyright 2018 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "cc/move.h"
+
+namespace minigo {
+
+std::ostream& operator<<(std::ostream& os, const Move& move) {
```

Comment:
I would probably expect `cout << move` to print a KGS/human-style coord rather than an SGF-style one, e.g. `W P16` vs `W[qq]`.  but i don't have any good solution for what to do with the << operator for making sgfs if we do that.  (no action needed, but made me go hmm)